docs/en/sql-reference/functions/splitting-merging-functions.md

```diff
@@ -403,3 +413,31 @@ Result:
 │ ['test1','test2','test3','test4'] │
 └───────────────────────────────────┘
 ```
+
+Using the ngram tokenizer with ngram length 3:
```

Comment:
Removed some of the examples to not overwhelm the reader (we should test all combinations in SQL tests but its not necessary to document all combinations in end user docs)